main.go

```diff
@@ -46,7 +46,12 @@ func main() {
 	fmt.Printf("\nExecuting Matrix...\n\n")
 	results := execute.Run(plan)
 
-	summary := output.Stream(results)
+	reporter, err := output.GetReporter(config.Report)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
I don't want the timestamp prefix'd for now, so I'm being lazy and doing it this way till I configure a logger :)